package.json

```diff
@@ -176,7 +176,7 @@
 				},
 				{
 					"command": "liberty.dev.start.container",
-					"when": "viewItem == libertyMavenProjectContainer || viewItem == libertyGradleProjectContainer",
+					"when": "viewItem == libertyMavenProject || viewItem == libertyGradleProject || viewItem == libertyMavenProjectContainer || viewItem == libertyGradleProjectContainer",
```

Comment:
`libertyMavenProjectContainer` & `libertyGradleProjectContainer` are specific types of Liberty Maven and Liberty Gradle projects. We actually do not want the "start in container" option to show for any Liberty Maven or Liberty Gradle project, so unfortunately this solution is not an option.